hugo/content/contribute.md

```diff
@@ -3,83 +3,239 @@ title: Contribute
-* Chat [Mumble room on Riot/Matrix](https://riot.im/app/#/room/#mumble:matrix.org) (shares messages with IRC)
-* Our [issue tracker](https://github.com/mumble-voip/mumble/issues)
+1. Take a look at existing [issues on GitHub](https://github.com/mumble-voip/mumble/issues) to check whether your issue or request was already reported.
+   If that's the case, then vote for it with a thumps up or add useful information & ideas with a comment.
+   
+2. If there is no issue report yet, then create a [new one](https://github.com/mumble-voip/mumble/issues/new/choose).
 
-# Report Bugs and Issues
 
-If you found a bug or issue you can report them in our [issue tracker](https://github.com/mumble-voip/mumble/issues). Please check if a ticket already exists for your issue before creating a new one.
+# Development
 
-# Suggest Changes and Features
+## Coding
 
-The issue tracker is also used to track [feature requests, which are marked with the `feature-request` label](https://github.com/mumble-voip/mumble/issues?q=is%3Aopen+is%3Aissue+label%3Afeature-request+sort%3Areactions-%2B1-desc).
+We are always looking for helping hands willing to contribute to Mumble.
 
-Please vote with a thumbs up on those you would like to see being fulfilled, describe your use cases or otherwise participate in the discussion on existing tickets. And if you want to request a feature not suggested yet, please create a new feature request ticket possibly with some elaboration and use cases.
+Mumble is mostly written in **C++** using the **Qt** library.
+Thus previous knowledge in C++ and Qt is definitely useful, but we also encourage interested people who want to learn it to give it a try. We're happy to help you out, if needed.
 
-# Test new features and developments
+Our source code is hosted on GitHub:
 
-Apart from our stable releases we also provide development snapshots for testing the current, intermediate state of development. You can download them from our [Downloads page]({{< relref "downloads" >}})).
+- [Overview of Mumble Projects/Repos](https://github.com/mumble-voip/)
 
-If you find any issues with them please check if the issue has already been reported, add details if you have more, or add a new tracker item if it has not been reported yet.
+- [Main Mumble Repo](https://github.com/mumble-voip/mumble): includes the Server and Client
+  
+- [iOS app](https://github.com/mumble-voip/mumble-iphoneos): unmaintained; help wanted
 
-# Translation
+### Getting Started
 
-Mumble currently (April 2020) supports over 15 different languages (38 if including unfinished translations). As more people get to know and use Mumble, versions in their native languages become more and more important.
+The best way to get into Mumble development is to link up with us on IRC ([#mumble-dev on freenode](irc://chat.freenode.net/mumble-dev)), also available via Matrix (Join: `#freenode_#mumble-dev:matrix.org` or use [Direct Link via Element.io (Web App)](#freenode_#mumble-dev:matrix.org)).
```

Comment:
Could someone test the Matrix method with:
`#freenode_#mumble-dev:matrix.org` ?

Reply:
Using `/join #freenode_#mumble-dev:matrix.org` seems to work

Reply:
Very good, then I assume this as resolved.